packages/storybook/src/nimble/tabs/tabs-matrix.stories.ts

```diff
@@ -28,12 +28,17 @@ const metadata: Meta = {
 
 export default metadata;
 
+const widthStates = ['', '250px'] as const;
+type WidthState = (typeof widthStates)[number];
+
 // prettier-ignore
 const component = (
     toolbar: TabsToolbarState,
-    [disabledName, disabled]: DisabledState
+    [disabledName, disabled]: DisabledState,
+    widthValue: WidthState
+
 ): ViewTemplate => html`
-    <${tabsTag} style="padding: 15px;">
+    <${tabsTag} style="padding: 15px; width: ${() => (widthValue ?? '')};">
```

Comment:
This will render out as invalid / incomplete css, ie: `style="width: "` can we make it render out correctly for either case?